### PR TITLE
Add show_command playbook

### PR DESCRIPTION
**Pull request by gdykeman**
Modification to stay agnostic.  End user will be prompted on launch for a limit to apply the correct show command to correct OS

### CODE REVIEW

File: network_commands.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+- name: GATHER INFORMATION FROM ROUTERS
+  hosts: all
+  gather_facts: no
+  connection: network_cli
+
+  tasks:
+    - name: SHOW COMMAND
```
Comment (IPvSean) [inline network_commands.yml]: nope, use cli_command

**Comment by gdykeman** [comment]
Please accept this is mission critical

**Comment by IPvSean** [comment]
maybe i will, maybe i won't

**Comment by IPvSean** [comment]
looks good 
look ok

**Pull request merged**